src/shell/command_helper.h

```diff
@@ -271,6 +283,82 @@ inline int compute_ttl_seconds(uint32_t expire_ts_seconds, bool &ts_expired)
     return 0;
 }
 
+inline void batch_execute_multi_set(scan_data_context *context)
+{
+    for (auto it = context->multi_kvs.begin(); it != context->multi_kvs.end(); ++it) {
```

Comment:
```suggestion
    for (const auto& it : context->multi_kvs) {
```

Reply:
for (const auto& kv : context->multi_kvs) {